src/foam/box/pipeline/PipelineNode.js

```diff
@@ -59,7 +59,7 @@ foam.CLASS({
       name: 'errorBox',
       documentation: 'Error box for RPC-related errors external to runnable.',
       factory: function() {
-        return this.LogBox.create({
+        return this.defaultErrorBox || this.LogBox.create({
```

Comment:
Just a heads up: this triggers a warning (`Access missing import: defaultErrorBox`) when `defaultErrorBox` is not provided. Is there a way to prevent the warning from showing up?